pkg/reconciler/v1alpha1/taskrun/taskrun.go

```diff
@@ -82,6 +83,58 @@ const (
 	imageDigestExporterContainerName = "step-image-digest-exporter"
 )
 
+type TaskRunWithTask struct {
```

Comment:
I got a bit confused about what this type was - it seems to me like the main goal of this type is to sort StepStates, how about calling it something like `StepStateSorter`?

Reply:
Done. Thx.